apps/ledger-live-mobile/src/screens/SyncOnboarding/AllowManagerDrawer.tsx

```diff
@@ -42,7 +42,7 @@ const AllowManagerDrawer = ({ isOpen, device, onClose }: Props) => {
       noCloseButton
     >
       <Flex mt={8} alignItems="center" justifyContent="center">
-        <Text fontWeight="semiBold" fontSize={24} textAlign="center">
+        <Text variant="h4" textAlign="center">
```

Comment:
didn't test but double check if you shouldn't use `h4Inter` instead (this one might be the big ass capitalized font)

Reply:
I don't think i can use `h4Inter` variant, only `h1Inter` seems to exist. But anyway `h4` is not a big ass capitalized font (`h3` is though)

Reply:
ah my bad I'm dumb I was checking react-ui 😆